pkg/tfvars/openstack/openstack.go

```diff
@@ -57,11 +59,26 @@ func TFVars(masterConfig *v1alpha1.OpenstackProviderSpec, cloud string, external
 	cfg.BaseImageName = imageName
 	if isURL {
 		// Valid URL -> use baseImage as a URL that will be used to create new Glance image with name "<infraID>-rhcos".
-		localFilePath, err := cache.DownloadImageFile(baseImage)
+		var localFilePath string
+
+		url, err := url.Parse(baseImage)
 		if err != nil {
 			return nil, err
 		}
 
+		// We support 'http(s)' and 'file' schemes. If the scheme is http(s), then we will upload a file from that
+		// location. Otherwise will take local file path from the URL.
+		if url.Scheme == "http" || url.Scheme == "https" {
```

Comment:
I think switch is more appropriate here..

Reply:
agree, will be done